lib/ueberauth/strategy/google.ex

```diff
@@ -81,87 +97,49 @@ defmodule Ueberauth.Strategy.Google do
   """
   def credentials(conn) do
     token = conn.private.google_token
-    scope_string = token.other_params["scope"] || ""
-    scopes = String.split(scope_string, " ")
-
-    %Credentials{
-      expires: !!token.expires_at,
-      expires_at: token.expires_at,
-      scopes: scopes,
-      token_type: Map.get(token, :token_type),
-      refresh_token: token.refresh_token,
-      token: token.access_token
-    }
+    credentials = UeberauthOidcc.Auth.credentials(token)
+    %{credentials | other: %{}}
   end
 
   @doc """
   Fetches the fields to populate the info section of the `Ueberauth.Auth` struct.
   """
   def info(conn) do
+    token = conn.private.google_token
     user = conn.private.google_user
 
-    %Info{
-      email: user["email"],
-      first_name: user["given_name"],
-      image: user["picture"],
-      last_name: user["family_name"],
-      name: user["name"],
-      birthday: user["birthday"],
-      urls: %{
-        profile: user["profile"],
-        website: user["hd"]
-      }
+    info = UeberauthOidcc.Auth.info(token, user)
+
+    %{
```

Comment:
Should this return `%Info{}`?

Reply:
It does. [UeberauthOidcc.Auth.info/2](https://hexdocs.pm/ueberauth_oidcc/UeberauthOidcc.Auth.html#info/2) returns an `Ueberauth.Auth.Info.t()` struct, and then the struct update `%{info | ...}` keeps it a struct but updates some of the values.

Reply:
I love it, sorry ... I missed that